pypit/arparse.py

```diff
@@ -3639,7 +3677,7 @@ def trace_number(self, v):
         self.update(v)
 
 
-class ARMLSD(BaseArgFlag):
+class ARMS(BaseArgFlag):
```

Comment:
While I think of it... I was unable to see if there's anywhere in the docs where we have specified what the acronym ARMS stands for (I did see it in `arms.py`). We should probably include this somewhere so that it's a bit more obvious to the user!!

Reply:
not sure where to add this..